services/skipchain/skipchain.go

```diff
@@ -456,6 +461,20 @@ func (s *Service) lenSkipBlocks() int {
 	return len(s.SkipBlocks)
 }
 
+// saves the actual identity
+func (s *Service) save() {
```

Comment:
Hah - I wrote `GetPath` first, then I thought that you don't like the `Get`, but it might be clearer ;) 
It's a getter, the service should not be able to set the path!